src/main/java/usecases/War.java

```diff
@@ -0,0 +1,167 @@
+package usecases;
+
+import entities.Card;
+import presenters.console.Input;
+import presenters.console.Output;
+
+import java.util.Random;
+import java.util.ArrayList;
+import java.util.List;
+
+public class War extends GameTemplate {
+    public final ArrayList[] playingField = new ArrayList[]{new ArrayList<Card>(), new ArrayList<Card>()};
+    private final Input gameInput;
+    private final Output gameOutput;
+    private final static String[] Hierarchy = new String[]{"A", "2", "3", "4", "5", "6", "7", "8", "9", "10", "J", "Q", "K"};
+
+    /**
+     * Constructor for War. Note that usernames must be a List of length 2
+     *
+     * @param gameInput  A Game.Input object allowing for player input.
+     * @param gameOutput A Game.Output object allowing for output to the player.
+     */
+    public War(Input gameInput, Output gameOutput, List<String> usernames, UserManager userManager, Random random) {
```

Comment:
Consider adding a constructor that does not have a `Random` parameter; look at `CrazyEights` for an example